### PR TITLE
charts: initial commit for csi-driver-lvm

### PR DESCRIPTION
**Pull request by Vicente-Cheng**
Initial commit for csi-driver-lvm

### CODE REVIEW

**Review by bk201 — APPROVED** [review]

File: charts/harvester-csi-driver-lvm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.0-dev
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: v0.1.0
```
Comment (tserong) [inline charts/harvester-csi-driver-lvm/Chart.yaml:24]: Nit: I'd suggest putting this in quotes (some of our other charts do use quotes, some don't, but https://helm.sh/docs/topics/charts/#the-appversion-field says quotes are recommended)

File: charts/harvester-csi-driver-lvm/templates/csi.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+          successThreshold: 1
+          timeoutSeconds: 3
+        ports:
+        - containerPort: 9898
+          name: healthz
+          protocol: TCP
+        resources: {}
+        securityContext:
+          privileged: true
+        terminationMessagePath: /termination.log
```
Comment (tserong) [inline charts/harvester-csi-driver-lvm/templates/csi.yaml:86]: `terminationMessagePath` for node-driver-registrar and liveness-probe is `/dev/termination-log`, but this one for harvester-csi-driver-lvm-plugin is `/termination.log`. Should they all be the same?

**Review by tserong — APPROVED** [review]
LGTM (just a couple of little things)

File: charts/harvester-csi-driver-lvm/Chart.yaml (hunk 1 above)
Comment (Vicente-Cheng) [reply charts/harvester-csi-driver-lvm/Chart.yaml:24]: Thanks @tserong.
I will add quotes in the next update.

I would like to have the first release for developing the addon capability.

File: charts/harvester-csi-driver-lvm/templates/csi.yaml (hunk 2 above)
Comment (Vicente-Cheng) [reply charts/harvester-csi-driver-lvm/templates/csi.yaml:86]: Ditto. I thought that was the customized path, so we could keep it or make some changes in the next update.

**Pull request merged**